tests/datalab/test_cleanvision_integration.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
```

Comment:
where in this file is there any unit test of correct functionality? Need to have at least one unit test that serves as end-to-end check that functionality actually works as desired. Eg. have a toy dataset with 4 images, one dark, and show datalab flags only this one image with the dark issue

Reply:
This part is already checked in cleanvision repo tests. I didn't add it to avoid duplication.

Reply:
how can we be sure it's working correctly in datalab?   I don't follow the logic that because its checked in cleanvision repo, there is no way it got messed up in datalab, so I there needs to be at least one end-to-end test for datalab as well

Reply:
Added an end to end test for datalab as well. Check [here](https://github.com/cleanlab/cleanlab/pull/679/files#diff-810ef4854434604d9f6d7d0b885359d14c915bd387f037dfbcc8b1cc3343acafR38)